src/scripts/API/__mocks__/TV-maze.js

```diff
@@ -0,0 +1,7 @@
+class TvAPI {
+  getAllShows = () => [1, 2, 3, 4, 5, 6, 7]
+}
+const tv = new TvAPI();
+
+// eslint-disable-next-line import/prefer-default-export
```

Comment:
- [ ] Please remove this eslint disable line its best to fix the issue in the code.